### DISCUSSION TITLE
Permissions error despite being logged in

### DISCUSSION THREAD

**Discussion by AxonJ** [community/community#137433] [Actions] [answered]
### Select Topic Area

General

### Body

Struggling to push changes from my repository, im getting a permissions error despite being logged in with my GitHub account via the terminal. 
Anyone else run into this issue that can help me?

**Comment by MiguelCardoso19** [comment]
Have you generated a personal acess token?

**Reply by AxonJ** [reply]
Yes i did.

**Reply by MiguelCardoso19** [reply, marked as answer]
Use the token as your password when prompted during the push.

`git push https://<username>:<your-token>@github.com/<username>/<repo>.git`

**Reply by AxonJ** [reply]
Thanks, i will try it tomorrow
